src/cmds/backup.ts

```diff
@@ -0,0 +1,138 @@
+import yargs from 'yargs';
+import chalk from 'chalk';
+import { environmentConfigExists, getEnvironmentsConfig } from '../utils/environmentUtils';
+import { CleanService, ExportService, ImportService, ZipService } from '@kentico/kontent-backup-manager/_commonjs/src';
```

Comment:
This is my bad, I've released a new version of the tool to fix types path. Please update and change to "@kentico/kontent-backup-manager/"